src/library/basetracktablemodel.cpp

```diff
@@ -25,6 +26,8 @@ const mixxx::Logger kLogger("BaseTrackTableModel");
 
 const QString kEmptyString = QStringLiteral("");
 
+constexpr int kHeightOfCoverartTooltip = 150; // Height of the image for the tooltip in pixel
```

Comment:
on FullHD the 150px square is alright for me, could be a bit bigger, max 200 px.

can we make this proportional to the screen size / library font size / dpi / Qt scaling factor somehow?
those may reflect how 'large' the user perceives the UI on his hardware.
https://doc.qt.io/qt-5/qscreen.html#availableSize-prop

Reply:
I changed this. 150px looked good for me on my 1600x900 screen, so I kept the ratio. On a FullHD screen you should get 178px.
The resolution is taken from the primary screen. Do you know, how to get the screen, on which a widget is actually drawn?

Reply:
I think it should use the window size. We cannot ussr that mixxx is full screened.

Reply:
> I think it should use the window size. We cannot ussr that mixxx is full screened.
Can you give me an example, how do get a reference to the window in this part of the code? This function seems to have no relation to any Qt object.

Reply:
util/widgethelper.h/.cpp might already contain the required code. Please extend it for reusability.

Reply:
> 
> 
> util/widgethelper.h/.cpp might already contain the required code. Please extend it for reusability.

Thanks for the hint, but it doesn't solve the problem. The function getWindow needs a Qwidget as argument. And I've neither a QWindow nor a QWidget in this part of code.

Reply:
Is C++ maybe the wrong place to define the size of this coverart image?
Maybe the right way is, to write always 200px and scale it to the final image size using a QSS style.

Reply:
I tried several variants, but I found no better solution:
-I can't determine the QWindow -> Therefore I can read neither Window size nor resolution of the screen displaying this window
-QSS is not available in the Tooltips. Only the limited HTML subset: https://doc.qt.io/qt-5/richtext-html-subset.html
Therefore the actual solution is the best I could do.